### ISSUE TITLE
Коррекция показаний

### ISSUE THREAD

**Issue by booroondook-eng** [vooon/elehant-to-mqtt#13]
Собственно говоря, не проблема, а просьба расширения функционала.
Сначала поясню. На практике бывает так, что счетчик "Элехант" (это относится и к водяным, и к газовым в равной мере) устанавливается в дополнение к официальному - просто для удобства пользования (например, официальный счетчик механический бнз возможности удаленного снятия показаний и/или расположен в неудобном месте).
При этом естественно возникает ситуация, при которой показания  "Элеханта" не совпадают с показаниями официального счетчика (т.к. там уже накопился расход за предыдущее время), а передавать в службы ЖКХ нужно именно "официальные" показания.
Поэтому **предложение №1**: сделать возможность коррекции показаний, прибавляя (или отнимая - что маловероятно, но тем не менее) к истинным показаниям "Элеханта" некую фиксированную величину, определенную разницей между показаниями официального счетчика и "Элеханта" на момент его ввода в эксплуатацию. Эту величину можно задать например, через соответствующий новый параметр config.json.
Но это еще не всё. Как показывает опыт, счётчики считают по-разному, и показания "Элеханта" могут со временем сильно разойтись с официальными. Например, я в течение 10 дней мониторил у себя это дело и выяснил, что мой СГБД "отстает" от официального счётчика на 0,105 м³ в час - это двольно много. Выяснять, какой из счётчиков - механический или СГБД - считает "правильно", а какой "неправильно", нет смысла, потому что передавать показания в газовую компанию всё равно придется по официальному счетчику.
Отсюда  **предложение №2**: реализовать (опять же - через новый параметр в config.json) поправочный коэффициент для показаний "Элеханта".

**Comment by booroondook-eng** [comment]
Сам себе и отвечу. Дурацкая идея. Все вычиселения надо делать средствами "умного дома", или где там они еще используются.

**Comment by vooon** [comment]
Да, вносите поправку прямо в ядре умного дома. Для HA прямо в `value_template` это легко реализуется.
